### ISSUE TITLE
Update with newer Windows Server limitations

### ISSUE THREAD

**Issue by eponerine** [MicrosoftDocs/windowsserverdocs#2482]
Support for XXXXX references updates in 2016 from 2012 R2. With the addition of deduplication + ReFS and multiple new OS's having come out, can we have official guidance on supported deployments. 

For example - **Support for Large Volumes** - is there still a max limit of 64TB on a dedup'd volume? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ff6d5e62-0c6d-ae58-5b63-6b60e0a66c5d
* Version Independent ID: 95b917ce-481e-09d5-2112-1f73ff2c53e5
* Content: [What's New in Data Deduplication](https://docs.microsoft.com/en-us/windows-server/storage/data-deduplication/whats-new)
* Content Source: [WindowsServerDocs/storage/data-deduplication/whats-new.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/storage/data-deduplication/whats-new.md)
* Product: **windows-server-threshold**
* GitHub Login: @wmgries
* Microsoft Alias: **wgries**

**Comment by pintokumar1608** [comment]
Hi, Is this feature in any time will be available in win10? I think this will be a great thing happens to end users who don't have extra storage and resources to enhance their storage. Thanks

**Comment by illfated** [comment]
(ref. #1219 #5211 #3809)

**Comment by IngridAtMicrosoft** [comment]
Thanks for your dedication to our documentation. If we have been unable to review your issue in a timely manner, we sincerely apologize for the delayed response. We have captured the informtion you have provided for review. We are closing this issue for now, but if you feel that it's still a concern, please respond and let us know. If you determine another possible update to our documentation, please don't hesitate to reach out again. #please-close

**Comment by IngridAtMicrosoft** [comment]
Hi @eponerine Thank you for your response!

**Comment by dknappettmsft** [comment]
Thanks for the feedback.  The addition of ReFS support that arrived in Windows Server 2019 has already been added to this article since this issue was opened. 64TB is still the maximum limit on a dedup'd volume.

This article will still contain references to specific Windows Server releases (e.g. 2016 and 2012 R2) as it's a What's New article designed to capture those changes as a point in time.

I have pushed a few minor changes that should be visible later today for clarity.
